app/helpers/admin/editions_helper.rb

```diff
@@ -279,8 +283,8 @@ def translation_preview_links(edition)
     end
   end
 
-  def archiving_or_unpublishing(edition)
-    edition.unpublishing.unpublishing_reason_id == UnpublishingReason::Archived.id ? 'archiving' : 'unpublishing'
+  def withdrawal_or_unpublishing(edition)
+    edition.unpublishing.unpublishing_reason_id == UnpublishingReason::Withdrawn.id ? 'withdrawal' : 'unpublishing'
```

Comment:
I was going by the language used for `Archiving`. Either/or works. It's only minor.